### PR TITLE
[activitystream] Make DRF render more nicely

### PR DESCRIPTION
**Pull request by relrod**
- Make it so the URL insertion works (by providing "activitystream-list")
- Give a nicer page title

### CODE REVIEW

File: ansible_base/activitystream/urls.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
 from django.urls import include, path
+from rest_framework import routers
 
 from ansible_base.activitystream import views
-from ansible_base.lib.routers import AssociationResourceRouter
 
-router = AssociationResourceRouter()
+router = routers.SimpleRouter()
```
Comment (john-westcott-iv) [inline ansible_base/activitystream/urls.py:6]: Out of curosity, why the change here? Is there something we need to/can do better in `AssociationResourceRouter`?

**Review by john-westcott-iv — APPROVED** [review]

File: ansible_base/activitystream/urls.py (hunk 1 above)
Comment (relrod) [reply ansible_base/activitystream/urls.py:6]: It's more that I don't think we need the functionality of ARR here. I'm not actually telling it to associate anything, in the router config. So it doesn't buy us anything in this case.
